en/docs/webhooks.md

```diff
@@ -2,3 +2,40 @@
 
 !!! info
     This section will describe Webhooks, its concepts, and its usage concerns/concepts in Choreo.
+
+## Introduction
+
+A webhook directs information from one application to another in real-time and triggers the receiving application to execute a specific process in response.
+
+A project can use a webhook to receive information from another project or an external application in order to execute a process. A project can also trigger another project or an external application by sending information to it via a webhook.
+
+## Design webhooks
+
+### Prerequisites
+
+You need to create a project to contain the webhook.
+
+### Create webhooks
+
```

Comment:
I don't think we need to provide the step-by-step instructions. Instead we should tell which systems we support. For example, we support Github and Slack as event sources. Each of these event sources have different channels (we can list them. they are also listed in the UI too.)
When we have a tutorial for creating a webhook type component, we can simply link it from here.